webservice/0.0.2/variables.tf

```diff
@@ -19,39 +19,51 @@ variable "env" {
   default     = {}
 }
 # @group "Resources"
-# @label "CPU"
-# @options ["0.5","1","2"]
-variable "cpu" {
+# @label "CPU Request"
+variable "request_cpu" {
   type        = string
-  description = "CPU"
-  default = "0.5"
+  description = "CPU request. e.g. 0.5, 1, 2"
+  default     = "0.1"
 }
 # @group "Resources"
-# @label "Memory"
-# @options ["1Gi","2Gi","4Gi"]
-variable "memory" {
+# @label "Memory Request"
+variable "request_memory" {
   type        = string
-  description = "Memory"
-  default = "512Mi"
+  description = "Memory request. e.g. 1Gi, 2Gi, 4Gi"
+  default     = "128Mi"
+}
+# @group "Resources"
+# @label "CPU Limit"
+variable "limit_cpu" {
+  type        = string
+  description = "CPU limit. e.g. 0.5, 1, 2"
+  default     = "0.5"
```

Comment:
Let's default to unlimited for all the resources

Reply:
set default to null